src/components/Caption/Caption.vue

```diff
@@ -0,0 +1,57 @@
+<!--
+  - @copyright Copyright (c) 2021 Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @author Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<div role="heading" class="app-navigation-caption">
```

Comment:
I'm a bit confused by this. We already have a Caption component for the navigation.
Is this component navigation only? It's also missing documentation.

I think the name should be more straightforward, and maybe the component should be in a dedicated section of the docs? Like 'Text', where we could also put the highlight component as well? :thinking:

Reply:
> Is this component navigation only

It's used in both navigation and sidebar in Talk

> It's also missing documentation

Oopsss

> I think the name should be more straightforward, and maybe the component should be in a dedicated section of the docs? Like 'Text', where we could also put the highlight component as well? thinking

Not sure I get this, quick call?

Reply:
> Not sure I get this, quick call?

For transparency and posterity it would be cool to have the infos, decisions and so on here with the PR.

Reply:
@ma12-co what was the results of the call ? Can you summarize ?

Reply:
Waiting for @jancborchardt 
Discussion here #1751